content/klc/S6.2.adoc

```diff
@@ -1,9 +1,25 @@
 +++
-brief = "Component fields must be filled appropriately"
+brief = "Symbol and alias fields and metadata filled out as required"
 +++
 
 . *Reference* field is selected appropriately for the symbol and is _visible_
 . *Value* field contains the name of the symbol and is _visible_
-. *Footprint* field contains footprint link for atomic parts, and is empty for generic parts. It must be _invisible_
-. *Datasheet* field is left empty, and is _invisible_
-. The symbol contains no other custom fields
\ No newline at end of file
+. *Footprint* field contains footprint link for fully specified symbols and must be _invisible_
+.. The footprint field must be empty for generic symbols
+. *Datasheet* entry [1] is filled out, and is _invisible_
+. The symbol contains no other custom fields
+
+Additional documentation is provided via
```

Comment:
Please add a colon at the end of this line.

Reply:
done

Reply:
I see a comma, not a colon. I was shooting for path of least resistance before, but since this isn't closed I feel the wording is a bit awkward and would read better if `two other fields:` was added after `via`.

Reply:
Remember you might need to be a bit clearer with what you mean when writing with non native speakers. For example you could have added the symbol you were expecting to your comment (as the symbol itself in addition to its english name)

Reply:
You are right. I will try to be more clear in the future. There are a few things I could have done better.

Reply:
I imagine it is hard to remember that not everyone here is a native speaker as our knowledge of english is well enough to hide this fact most of the time.